ios/AllAboutOlaf.xcodeproj/project.pbxproj

```diff
@@ -49,6 +48,16 @@
 		3AE408501E1E280800F0FD83 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 3AE4084F1E1E280800F0FD83 /* LaunchScreen.storyboard */; };
 		832341BD1AAA6AB300B99B32 /* libRCTText.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 832341B51AAA6A8300B99B32 /* libRCTText.a */; };
 		BBDFD91C684046CB9FD0CD55 /* Entypo.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 742B362B86E443C09A7F9FD9 /* Entypo.ttf */; };
+		A3EA8C86662742928E156E45 /* libTouchID.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 905F5155B95042B89DE3FADA /* libTouchID.a */; };
+		2FE5DD35D6C74CBDAA79E5E9 /* EvilIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = E7F4C82AC8DF4ADEBE2ED75C /* EvilIcons.ttf */; };
+		911AFFD0515E4820870BD025 /* Feather.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 7578036A3F5A41D5ABC88414 /* Feather.ttf */; };
+		D314365E6D264BFC9EE8CFFE /* FontAwesome.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 0D4853F7133F4472B8EA09F6 /* FontAwesome.ttf */; };
+		46A7D3950A5A4F6193ED2E4E /* Foundation.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 4D07275C973147788D8776E7 /* Foundation.ttf */; };
+		A0198A761F4B421380FB751B /* MaterialCommunityIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 93C62C74F0F34814A515C299 /* MaterialCommunityIcons.ttf */; };
+		5445064DAFEF4267891C7732 /* MaterialIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 1A1797A4E0CD4A159B7E5221 /* MaterialIcons.ttf */; };
+		A9C2CF0EA1E54AEDA882B107 /* Octicons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 120227DBD4FE4E44AE8B829F /* Octicons.ttf */; };
+		4B899B1338F2467FBC64A25D /* SimpleLineIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = A236B5D5F384407EAC985D9B /* SimpleLineIcons.ttf */; };
```

Comment:
I'll need you to remove the fonts that got added to the Android folders, in addition to any added lines that reference the .ttf files in this .pbxproj.